examples/object_detection_3d/object_detection_3d/rotate_iou.py

```diff
@@ -0,0 +1,387 @@
+# Copyright 2021-2023 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Copied from https://github.com/open-mmlab/mmdetection3d
+# Licensed under Apache-2.0
+# Copyright (c) OpenMMLab. All rights reserved.
+#####################
+# Based on https://github.com/hongzhenwang/RRPN-revise
+# Licensed under The MIT License
+# Author: yanyan, scrin@foxmail.com
+#####################
+import math
+
+import numba
+import numpy as np
+from numba import cuda
```

Comment:
Just noticed this would prevent running the codes in CI/CD. If there's no quick alternatives, will have to drop testing for this example workflow.

Reply:
Are you able to run with CUDA on your laptop?

Reply:
My laptop is M1, cannot run CUDA-specific codes. However, I ran the codes in a windows machine that has CUDA to create the preview test-suite/results as linked in the PR description.